primitives/src/merkle_tree/internal.rs

```diff
@@ -544,73 +544,105 @@ where
     }
 
     /// Update the element at the given index.
-    pub(crate) fn update_internal<H, Arity>(
+    pub(crate) fn update_with_internal<H, Arity, F>(
         &mut self,
         height: usize,
         pos: impl Borrow<I>,
         traversal_path: &[usize],
-        elem: impl Borrow<E>,
-    ) -> Result<LookupResult<E, (), ()>, PrimitivesError>
+        f: F,
+    ) -> Result<(i64, LookupResult<E, (), ()>), PrimitivesError>
```

Comment:
Document what this `i64` is

Reply:
Documented in [937c573](https://github.com/EspressoSystems/jellyfish/pull/475/commits/937c57379424b074bfe5caf61d3b7538a8b8deb8)